gapir/replay_service/service.proto

```diff
@@ -86,12 +88,14 @@ message PostData {
 // build time of the replay instruction.
 message Notification {
```

Comment:
The general rule for protobuf is that you never re-number fields. You only add to the end.
However since we ship the client&server in one package, this won't actually be a problem.